update_copyrights

```diff
@@ -103,28 +103,29 @@ for root, _, files in os.walk(options.repo):
     for filename in files:
         filetype = Path(filename).suffix
         if filetype == ".png" or filetype == ".jpg" or filetype == ".webp" or filetype == ".wav" or filetype == ".ogg":
-            file = os.path.normpath(os.path.join(root, filename))
-            hash = do_hash(file)
-
-            if not file in csv_data:
-                added.append(["", file, "", "", "", do_git(file), hash])
-            elif len(csv_data[file]) < 7:
-                missing_fields.append(csv_data[file])
-            elif len(csv_data[file]) > 7:
-                extra_fields.append(csv_data[file])
-            elif csv_data[file][5] != "":
-                update.append(csv_data[file])
-            elif csv_data[file][6] != hash:
-                csv_data[file][5] = do_git(file)
-                csv_data[file][6] = hash
-                changed.append(csv_data[file])
-            elif csv_data[file][2].strip() == "" or csv_data[file][3].strip() == "":
-                incomplete.append(csv_data[file])
-            elif not csv_data[file][2] in known_licenses:
-                unknown_licenses.append(csv_data[file][2])
-                incomplete.append(csv_data[file])
+            # Always use slashes for the file path irrespective of OS used to run the update
+            file_path = os.path.normpath(os.path.join(root, filename)).replace('\\', '/')
```

Comment:
Using `posixpath` instead of `os.path` should handle the slashes without needing the `replace`. I don't have a Windows PC to test that on though.

Reply:
I experimented with `pathlib.PurePosixPath` on Windows, but it didn't seem to work for me. I think it's the `normpath` call on Windows that produces the backslashes.

Reply:
What does `posixpath.normpath` do?

I'm not suggesting `import pathlib.PurePosixPath`, I'm suggesting `import posixpath`.

Reply:
I'd never even heard of `posixpath`. Opening commentary in https://github.com/python/cpython/blob/3.12/Lib/posixpath.py suggests it's not intended to be imported directly. Even if `posixpath.normpath` achieves what you expect, should we really be relying on internal implementations?

Reply:
On both Windows and Linux, `posixpath.normpath('fred\\fred.txt')` returns `'fred\\fred.txt'`.

Reply:
Importing it directly is documented in the second grey box of https://docs.python.org/3/library/os.path.html

However, I confirm that `posixpath.normpath` doesn't do what we want, even if it's given an `ntpath` instead of an arbitrary string. :(

Reply:
A string replace looks bad but since it's after `normpath` it should be correct if you use `os.path.sep`. Replacing backslash on POSIX systems is wrong.

Reply:
I didn't think a backslash would be possible for a normalised POSIX path. Plus `os.path.sep` is dependent on the OS. Are you saying you prefer `.replace(os.path.sep, '/')`?

Reply:
Yes.

A file or dir name can contain anything but slash and NUL in POSIX.